src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -134,6 +134,13 @@ public SBApiResponse profileUpdate(Map<String, Object> request, String userToken
 				return response;
 			}
 			Map<String, Object> profileDetailsMap = (Map<String, Object>) requestData.get(Constants.PROFILE_DETAILS);
+			String validatePhoneEmail = validateExistingPhoneEmail(profileDetailsMap);
+			if (validatePhoneEmail.equals(Constants.EMAIL_EXIST_ERROR) || validatePhoneEmail.equals(Constants.PHONE_NUMBER_EXIST_ERROR)) {
```

Comment:
validatePhoneEmail -- rename to errMsg 
Just check string is not empty... if not empty return bad_request...

Reply:
have added check to validate the key should be present else it will return appropriate message